### PR TITLE
REST-243: Rename files of trainingPegasus to DotnetBoilerPlate

### PR DESCRIPTION
**Pull request by PrayatOsm**
For new project we have replaced every occurance of trainingPegasus to DotnetboilerPlate

- All folder names
- All file names
- All namespaces
- All using directives
- Solution name
- Along with this we have added references for the new csproj files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,7 +33,7 @@ bld/
 [Ll]ogs/
 
 # Visual Studio 2015/2017 cache/options directory
-.vs/
+DotnetBoilerPlate/.vs
```
Comment (sameer-s-b) [inline .gitignore]: Rename the project to DotnetFoundation

File: DotnetBoilerPlate/DotnetBoilerPlate.DAL/Models/CandidateExamsDBO.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+namespace DotnetBoilerPlate.DAL.Models;
+
+public class CandidateExamsDBO
```
Comment (sameer-s-b) [inline DotnetBoilerPlate/DotnetBoilerPlate.DAL/Models/CandidateExamsDBO.cs]: We will not need this database object. We are trying to build a boilerplate app that has common work items that can be reused by copying this project content and building on top of it. So, not just the name, but anything that is related to Training pegasus will have to be removed.

Comment (sameer-s-b) [reply DotnetBoilerPlate/DotnetBoilerPlate.DAL/Models/CandidateExamsDBO.cs]: We should use Asp.Net Identity to setup logins/signups/roles. That will be a task on it's own.

File: .gitignore (hunk 1 above)
Comment (PrayatOsm) [reply .gitignore]: Renamed all files names to DotnetFoundation

File: DotnetBoilerPlate/DotnetBoilerPlate.DAL/Models/CandidateExamsDBO.cs (hunk 2 above)
Comment (PrayatOsm) [reply DotnetBoilerPlate/DotnetBoilerPlate.DAL/Models/CandidateExamsDBO.cs]: Removed all DBOs.Kept only Users files

**Review by sameer-s-b — APPROVED** [review]

**Pull request merged**